src/Tribe/Assets.php

```diff
@@ -99,7 +99,11 @@ public function admin_enqueue_scripts() {
 				[ 'event-tickets-attendees-list-js', 'attendees-list.js', [ 'jquery' ] ],
 				[ 'event-tickets-admin-accordion-js', 'accordion.js', [] ],
 				[ 'event-tickets-admin-accordion-css', 'accordion.css', [] ],
-				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ] ],
+				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ], 'admin_enqueue_scripts',
+				    [
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Multi-line array item not aligned correctly; expected 16 spaces, but found 20

<sub>hash: 6f53c68598c83b7d79a9987c54ebd2a2</sub>